internal/utils/generate_scripts.go

```diff
@@ -0,0 +1,188 @@
+package utils
+
+import (
+	"os"
+	"text/template"
+
+	"github.com/NethermindEth/1Click/configs"
+	"github.com/NethermindEth/1Click/internal/templates"
+	log "github.com/sirupsen/logrus"
+)
+
+/*
+GenerateDockerComposeScripts :
+This function is responsible for generating docker-compose scripts for execution, consensus and
+validator clients.
+
+params :-
+a. executionClient string
+Execution client whose script is to be generated
+b. consensusClient string
+Execution client whose script is to be generated
+c. validatorClient string
+Execution client whose script is to be generated
+d. path string
+Path of generated scripts
+
+returns :-
+None
+*/
+func GenerateDockerComposeScripts(executionClient, consensusClient, validatorClient, path string) {
+	_, baseTmp := buildTemplate("docker-compose_base", "base")
```

Comment:
buildTemplate returns (tmp *template.Template, rawTmp string)
and you are not using the *template.Template so then why use this function ?
Effectively you just need the template in string so create another function for it.